src/LibChorus.TestUtilities/RepositorySetup.cs

```diff
@@ -219,9 +217,11 @@ public static void MakeRepositoryForTest(string newRepositoryPath, string userId
 			hg.SetUserNameInIni(userId,  progress);
 		}
 
-		public static string ProjectName
+		public string ProjectName { get; private set; }
+
+		public static string ProjectNameForTest
 		{
-			get { return "ไก่ projéct"; } //nb: important that it have a space, as this helps catch failure to enclose in quotes
```

Comment:
With this change, several tests that were essentially testing the unicode path will now instead be testing the CP1252 path. If you're happy with that, I am too, just pointing it out.